lib/fake_ifdh.py

```diff
@@ -253,6 +253,8 @@ def chmod(dest: str, mode: int) -> None:
         os.chmod(dest, mode)
     except FileNotFoundError as e:
         pass
+    except PermissionError as e:
```

Comment:
Nitpicky - do we ever expect to handle these two exceptions differently?  If not, we should probably make this `except (FileNotFoundError, PermissionError) as e`.  But if we think we might expand the exception handling, leave it as is.

Either way, LGTM

Reply:
I think we might want to do something more interesting here later; ignoring it entirely is a broad brush... but for now I'm not thinking of any specific cases we want to handle differently...